src/CodeReviews.coffee

```diff
@@ -137,14 +137,20 @@ class CodeReviews
   # @param String origin_room string of origin room
   # @param String channel_to_notify string of the user/room to notify
   notify_channel: (cr, origin_room, channel_to_notify) ->
-    attachments = []
-    attachments.push
-      fallback: "#{cr.url} could use your :eyes: Remember to claim it in ##{origin_room}"
-      text: "*<#{cr.url}|#{cr.slug}>* could use your :eyes: Remember to claim it" +
-      " in <https://slack.com/app_redirect?channel=#{origin_room}|##{origin_room}>"
-      mrkdwn_in: ["text"]
-      color: "#575757"
-    sendFancyMessage @robot, channel_to_notify, attachments
+    slackTeamId(@robot).then (teamId) ->
```

Comment:
```suggestion
    slackTeamId(@robot).then (teamId) =>
```

We'll need the equals > instead of the thinner -> because we need to pass along @robot appropriately